tests/providers/sftp/sensors/test_sftp.py

```diff
@@ -97,11 +97,12 @@ def test_file_not_new_enough(self, sftp_hook_mock):
         sftp_hook_mock.return_value.get_mod_time.assert_called_once_with("/path/to/file/1970-01-01.txt")
         assert not output
 
+    @pytest.mark.parametrize("newer_than", (datetime(2020, 1, 2), "2020-01-02 00:00:00+00:00"))
```

Comment:
Small nit, "2020-01-02 00:00:00+00:00" - doesn't look as naive datetime

Reply:
Doesn't look but ... It's actually the str representation of "datetime(2020, 1, 2)" like it could be passed as a jina template in newer_than.

Reply:
- datetime(2020, 1, 2) parsed as a tz-naive datetime, and required to convert it to the aware again, which is not as trivial as it might seem
- `"2020-01-02 00:00:00+00:00"` as tz-aware datetime and already has timezone information

`datetime(2020, 1, 2) != datetime(2020, 1, 2, tzinfo=timezone.utc)`

Reply:
Rename and add test case parameters.